### ISSUE TITLE
Error while running  tff server

### ISSUE THREAD

**Issue by raghu20ram** [PAIR-code/federated-learning#50]
Hi , this is a great work. I have tried running the server using the below code 
`
import * as http from 'http';
import * as federated from 'federated-learning-server';

const INIT_MODEL = 'file:///initial/model.json';
const webServer = http.createServer(); // can also use https
const fedServer = new federated.Server(webServer, INIT_MODEL);

fedServer.setup().then(() => {
  webServer.listen(80);
});
`
I have passed my pretrained keras model which was converted to tfjs. I am facing the below error 
`
(node:8028) UnhandledPromiseRejectionWarning: Error: Unknown loss categoricalCrossEntropy
    at new ValueError (D:\Projects\federated-learning\src\server\node_modules\@tensorflow\tfjs-layers\dist\errors.js:36:28)
`
Can you please help out!. TIA
